observability/otel/otel-collector/otel-collector-processors.adoc

```diff
@@ -430,3 +430,26 @@ config: |
 <2> Defines a value provided in the `metrics` field as a `strict` exact match or `regexp` regular expression.
 <3> Lists the metric names, which are exact matches or matches for regular expressions, of the metrics to be converted to delta metrics. If a metric matches both the `include` and `exclude` filters, the `exclude` filter takes precedence.
 <4> Optional: Configures which metrics to exclude. When omitted, no metrics are excluded from conversion to delta metrics.
+
+[id="groupbyattrsprocessor-processor_{context}"]
+== Group-by-Attributes Processor
+
+This processor groups all spans, log records, and metric datapoints that share the same attributes by reassigning them to a Resource that matches those attributes.
```

Comment:
```suggestion
The Group-by-Attributes Processor groups all spans, log records, and metric datapoints that share the same attributes by reassigning them to a Resource that matches those attributes.
```
All the other processors seem to start this way (barring the exception of Cumulative to Delta Processor).

Reply:
Updated. Thank you.